src/data/migrations/20180930225846-create-change-tracking.js

```diff
@@ -58,6 +58,10 @@ module.exports = {
         field: 'iofog_uuid',
         references: { model: 'Fogs', key: 'uuid' },
         onDelete: 'cascade'
+      },
```

Comment:
Same as above. New migration needed. But all your changes can be merged into one single migration.

Reply:
Updated